src/main/java/com/denizenscript/ddiscordbot/events/DiscordInviteCreateScriptEvent.java

```diff
@@ -0,0 +1,71 @@
+package com.denizenscript.ddiscordbot.events;
+
+import com.denizenscript.ddiscordbot.DiscordScriptEvent;
+import com.denizenscript.ddiscordbot.objects.DiscordChannelTag;
+import com.denizenscript.ddiscordbot.objects.DiscordGroupTag;
+import com.denizenscript.ddiscordbot.objects.DiscordUserTag;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import net.dv8tion.jda.api.events.guild.invite.GuildInviteCreateEvent;
+
+public class DiscordInviteCreateScriptEvent extends DiscordScriptEvent {
+
+    // <--[event]
+    // @Events
+    // discord invitation created
+    //
+    // @Switch for:<bot> to only process the event for a specified Discord bot.
+    // @Switch channel:<channel_id> to only process the event for a specified Discord channel.
+    // @Switch group:<group_id> to only process the event for a specified Discord group.
+    //
+    // @Triggers when a Discord user creates an invitation.
+    //
+    // @Plugin dDiscordBot
+    //
+    // @Group Discord
+    //
+    // @Context
+    // <context.bot> returns the relevant DiscordBotTag.
+    // <context.group> returns the DiscordGroupTag.
+    // <context.channel> returns the DiscordChannelTag.
```

Comment:
I'd provide context here for at least the channel (and maybe also the group?), as it's not super clear what is it on first read - I assume the channel the invite leads to?
Otherwise LGTM

Reply:
Done 👍🏻